docs/ai_actions/work_with_ai_actions.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Create new AI actions or modify existing ones to work faster and increase creativity.
+---
+
+# Work with AI actions
+
+AI actions define what results are available to editors in AI-enabled areas, such as, for example, the AI Assistant.
+If the AI Actions LTS update has been installed and configured in your application, and you have administrator permissions, you can reconfigure the existing AI actions, and create new ones.
+
+## View AI actions
+
+With administrator permissions, you can view all AI actions configured in the application by navigating to the Admin Panel and selecting **AI actions**.
```

Comment:
Same as above about "required permissions".

Reply:
We've used this trick before, because only administrators can access the Admin menu, but you are right, you'd need admin rights and action/view.